microarray/src/org/labkey/microarray/MicroarrayModule.java

```diff
@@ -50,38 +43,21 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
-import java.util.List;
 import java.util.Set;
 
 public class MicroarrayModule extends SpringModule
 {
-    public static final String NAME = "Microarray";
-    public static final String WEBPART_MICROARRAY_RUNS = "Microarray Runs";
-    public static final String WEBPART_MICROARRAY_STATISTICS = "Microarray Summary";
-    public static final String WEBPART_PENDING_FILES = "Pending MAGE-ML Files";
     private static final String WEBPART_FEATURE_ANNOTATION_SET = "Feature Annotation Sets";
 
     private static final String CONTROLLER_NAME = "microarray";
     private static final String FEATURE_ANNOTATION_SET_CONTROLLER_NAME = "feature-annotationset";
 
     public static final String DB_SCHEMA_NAME = "microarray";
 
-    public static final AssayDataType MAGE_ML_INPUT_TYPE =
-            new AssayDataType("MicroarrayAssayData", new FileType(Arrays.asList("_MAGEML.xml", "MAGE-ML.xml", ".mage"), "_MAGEML.xml"), "MageML");
-    public static final AssayDataType TIFF_INPUT_TYPE =
-            new AssayDataType("MicroarrayTIFF", new FileType(Arrays.asList(".tiff", ".tif"), ".tiff"), "Image");
     public static final AssayDataType QC_REPORT_INPUT_TYPE =
             new AssayDataType("MicroarrayQCData", new FileType(".pdf"), "QCReport");
     public static final AssayDataType THUMBNAIL_INPUT_TYPE =
             new AssayDataType("MicroarrayImageData", new FileType(".jpg"), "ThumbnailImage");
-    public static final AssayDataType FEATURES_INPUT_TYPE =
-            new AssayDataType("MicroarrayFeaturesData", new FileType("_feat.csv"), "Features");
-    public static final AssayDataType GRID_INPUT_TYPE =
-            new AssayDataType("MicroarrayGridData", new FileType("_grid.csv"), "Grid");
-
-    /** Collection of all of the non-MageML input types that are handled specially in the code */
-    public static final List<AssayDataType> RELATED_INPUT_TYPES =
-            Arrays.asList(QC_REPORT_INPUT_TYPE, THUMBNAIL_INPUT_TYPE, FEATURES_INPUT_TYPE, GRID_INPUT_TYPE);

```

Comment:
Shouldn't getVersion() return 19.21?

Reply:
Ah, I misread this bit in the docs. Thank you, updated!